namespaces/live-1.cloud-platform.service.justice.gov.uk/formbuilder-monitoring/resources/main.tf

```diff
@@ -0,0 +1,13 @@
+terraform {
+  backend "s3" {
+  }
+}
+
+provider "aws" {
+  region = "eu-west-2"
+}
+
+provider "aws" {
+  alias  = "london"
+  region = "eu-west-2"
+}
```

Comment:
@mogaal Do we need to have two definitions of the aws provider if they're both for the same region? This seems to have been done in other `main.tf` files